endpoints/openrtb2/sample-requests/custom-currency/valid/non-usd-bid.json

```diff
@@ -0,0 +1,66 @@
+{
+  "description": "USD BidRequest gets converted because mockbidder bids in foreign currency, custom conversion rate is used",
```

Comment:
Isn't `non-usd-bid.json` more descriptive? This is the sole JSON test where the `mockBidder` bids in a currency other than US dollars.

Reply:
The reason I suggested something like `reverse-currency-conversion` is because that is the functionality we're trying to test. I agree that this is the only test that where the `mockBidder` bids in a currency that is not USD but ultimately `USD` is just one of the different currencies that we could use for these tests and with this name for someone to understand what this is exactly testing, we're expecting them to have the additional context that all other tests are bidding only in `USD`. If they don't have that additional context, it will be hard for them to understand what we're testing in this file based on the filename. We should try to name the tests as close to the functionality that we're testing as possible.

Reply:
Sorry I missed this comment Mansi. Done